### ISSUE TITLE
Empty directories

### ISSUE THREAD

**Issue by gvanem** [openssl/openssl#12815]
I've noted that when build OpenSSL using MSVC, some empty directories are created:
```
  ./crypto/include
  ./crypto/providers
```

I configured with:
```
  perl.exe -I. -I./util/perl Configure VC-WIN32-ONECORE no-makedepend \
  enable-zlib enable-trace enable-md2 no-ec_nistp_64_gcc_128 \
  enable-asan enable-aria enable-rc5 enable-heartbeats --prefix=/foo/bar --openssldir=/foo/bar
```

I see that the generated `Makefile` has lots of `-I "crypto\include"`. AFAICS needlessly.<br>
But I see no `mkdir crypto\include` anywhere. And I don't see `mkdir-p.pl` is used to create these. 

I mailed about this issue [here](https://mta.openssl.org/pipermail/openssl-users/2020-September/012956.html) also.  Quoting from Dr Mathias [reply](https://mta.openssl.org/pipermail/openssl-users/2020-September/012957.html):

>  it seems like the empty directories are created by the Configure script. 
> ...

Caused by these https://github.com/openssl/openssl/commit/25f2138b0ab54a65ba713c093ca3734d88f7cb51 and https://github.com/openssl/openssl/pull/9333

**Comment by levitte** [comment]
Nice catch.  Fixed in #12816, please try it out

**Comment by gvanem** [comment]
> please try it out

Done that. Seems fine now.

> levitte linked a pull request that will close this issue 2 hours ago

What English is this Github? Maybe "*levitte linked a pull request 2 hours ago that will close this issue*" (or *could*?)

**Comment by mspncp** [comment]
> > levitte linked a pull request that will close this issue 2 hours ago
> 
> What English is this Github? Maybe "_levitte linked a pull request 2 hours ago that will close this issue_" (or _could_?)

It sounds incredibly stupid on first reed, but it actually makes sense, if you already know what it means: By adding a `Fixes #12815` annotation to the commit message and/or pull request description, @levitte has indicated to GitHub that pull request #12816 will fix this issue. Now the pull request is linked to this issue, and when it gets merged, GitHub will automatically close the issue.

**Comment by mspncp** [comment]
In particular, "2 hours ago" refers to "linked a pull request" and not to "close this issue" :rofl:
